### ISSUE TITLE
ARIA 1.2 support/adoption in the components 

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental-styles#4955]
- [ ] formal compliancy
- [ ] enhancements for better ACC support
- [ ] identify tasks

`FIORITECHP1-28351`

**Comment by InnaAtanasova** [comment]
This should be part of the wcag and design updates and be done component per component. Can't be one ticket and assigned to one person.
